docs/index.md

```diff
@@ -35,6 +36,22 @@ Additionally, we prioritise the security of the data during its processing and t
 
 ![](_static/structure.png)
 
+### Contributions
+
+Besides our [models](https://www.foundationmodels.dk/models/). The Danish Foundation models project have led to contributions for a series of open-source projects, the following table include some of these contributions:
+
+
+| Project                                                                                                                | Contribution                                                  |
+| ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------- |
+| **Packages**                                                                                                           |                                                               |
+| [NLPDedup](https://github.com/saattrupdan/NLPDedup)                                                                    | A deduplication library derived from DFM's deduplication code |
+| **Code contributions**                                                                                                 |                                                               |
+| [TextDescriptives](https://hlasse.github.io/TextDescriptives/)                                                         | Added heuristic quality measure for texts                     |
+| [dolma](https://github.com/allenai/dolma)                                                                              | Bugfixes and addition of taggers for filtering                |
```

Comment:
@rlrs You could add your contribution to `transformers` here, significantly improving their Rust tokeniser speed.

Reply:
@rlrs if you have a reference I would love it add it